Godeps/_workspace/src/k8s.io/kubernetes/pkg/client/unversioned/clientcmd/loader.go

```diff
@@ -41,9 +42,23 @@ const (
 	RecommendedSchemaName       = "schema"
 )
 
-var OldRecommendedHomeFile = path.Join(os.Getenv("HOME"), "/.kube/.kubeconfig")
-var RecommendedHomeFile = path.Join(os.Getenv("HOME"), RecommendedHomeDir, RecommendedFileName)
-var RecommendedSchemaFile = path.Join(os.Getenv("HOME"), RecommendedHomeDir, RecommendedSchemaName)
+var RecommendedHomeFile = path.Join(HomeDir(), RecommendedHomeDir, RecommendedFileName)
+var RecommendedSchemaFile = path.Join(HomeDir(), RecommendedHomeDir, RecommendedSchemaName)
+
+// currentMigrationRules returns a map that holds the history of recommended home directories used in previous versions.
+// Any future changes to RecommendedHomeFile and related are expected to add a migration rule here, in order to make
+// sure existing config files are migrated to their new locations properly.
+func currentMigrationRules() map[string]string {
```

Comment:
I wasn't completely sure about the need of a migration for this, but we can always have users that were relying on an existing config and I wouldn't like to break things for them. If `oc` is the only reason you ever use the command line on Windows, it's likely that you won't change dirs and you could have a valid config already. Happened on my own Windows box.